flipperdevices/flipperzero-firmware
# [FL-2505] Active RPC session icon

# What's new

- RPC session icon for USB sessions

# Verification 

- Connect to qFlipper, check that the new icon is present
- Connect to the mobile app, check that the icon isn't present

# Checklist (For Reviewer)
